lib/peep.rb

```diff
@@ -0,0 +1,35 @@
+require 'pg'
+
+
+class Peep
+
+  attr_reader :message, :time
+  
+  def initialize(message:, time:)
+    
+    @message = message
+    @time = time
+  end
+
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
+    result = connection.exec("SELECT * FROM peeps ORDER BY id DESC")
```

Comment:
Good job calling ORDER by in the connection, that's a more elegant solution that I found